### DISCUSSION TITLE
Fermadata player white screen

### DISCUSSION THREAD

**Discussion by mohellebi-mounir** [AndreyPavlenko/Fermata#200] [General]
First of all, i would thank the developer for this awsome app!!! You did a great job!!!

I'm running the latest version on my android auto. And i remarque a strange behavior when i run a YouTube playlist. After three or four videos on full screen. The next vidéo sound is normally playing but no images. Only a white screen and nothing to do to come back to the main menu. The only solution is to stop android auto and restart it. Can you please try to fix this bug please?!!

**Comment by olegkr52** [comment]
I'm experiencing the same behavior. The screen becomes white and the menu button brings up empty frame. Pressing the back button gets out of full screen mode, but the navigation bar is no longer there. The only way to fix is to force stop Fermata in the phone.
Running Fermata 1.8.11 with AA 8.0.623354-release
Thank you!
